test/server/util/parse.test.ts

```diff
@@ -0,0 +1,18 @@
+import { extractShortUrl } from '../../../src/server/util/parse'
```

Comment:
If you're making a new test, could you kindly put it under `src/server/util/__tests__` instead please?

Reply:
shifted the test to the path